tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -1165,6 +1167,35 @@ public static function cacheConfigurationProvider(): array
         ];
     }
 
+    public function testEntityAttributeExcludesFromContainer()
+    {
+        if (! interface_exists(Entity::class)) {
+            self::markTestSkipped('This test requires ORM');
+        }
+
+        $container = $this->getContainer();
+        $extension = new DoctrineExtension();
+
+        $config = BundleConfigurationBuilder::createBuilder()
+            ->addBaseConnection()
+            ->addBaseEntityManager()
+            ->build();
+
+        $extension->load([$config], $container);
+
+        $attributes = $container->getAutoconfiguredAttributes();
+        $this->assertInstanceOf(Closure::class, $attributes[Entity::class]);
+
+        $reflector  = new ReflectionClass(TestCustomIdGeneratorEntity::class);
+        $definition = new ChildDefinition('');
+        $attribute  = $reflector->getAttributes(Entity::class)[0]->newInstance();
+
+        $attributes[Entity::class]($definition, $attribute);
```

Comment:
Not sure if this isn't unnecessarily convoluted, is there a harm in just doing
```suggestion
        $definition = new ChildDefinition('');
        $attributes[Entity::class]($definition, new Entity());
```
?

Reply:
Thanks for the review, I cleaned up the test.